conduit/plugins/exporters/postgresql/postgresql_exporter.go

```diff
@@ -112,10 +113,11 @@ func (exp *postgresqlExporter) Init(ctx context.Context, initProvider data.InitP
 	exp.ctx, exp.cf = context.WithCancel(ctx)
 	exp.logger = logger
 
-	db, ready, err := createIndexerDB(exp.logger, false, cfg)
+	db, ready, exporterCfg, err := createIndexerDB(exp.logger, false, cfg)
 	if err != nil {
 		return fmt.Errorf("db create error: %v", err)
 	}
+	exp.cfg = exporterCfg
```

Comment:
You could also put this right at the top and pass it into the createIndexerDB function:

```
	if err := cfg.UnmarshalConfig(&exp.cfg); err != nil {
		return fmt.Errorf("failure in unmarshalConfig: %v", err)
	}
	db, ready, err := createIndexerDB(exp.logger, false, exp.cfg)
```

alternatively, slight nit
```suggestion
	db, ready, exp.cfg, err := createIndexerDB(exp.logger, false, cfg)
	if err != nil {
		return fmt.Errorf("db create error: %v", err)
	}
```

Reply:
I need to revert this change because otherwise, I need to change the assignment to a non-creating assignment and declare the variables `db, ready, err` on separate lines.

<img width="1170" alt="image" src="https://github.com/algorand/conduit/assets/291133/8d1fd5e7-ba51-433b-b2c1-2ce8f91feb85">

Reply:
reverted in e69c231be21c9ed46e1f0f3b1afd1478d3cf4f43